### PR TITLE
Add Browser page

### PR DESCRIPTION
**Pull request by ikeith**

### CODE REVIEW

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
added to minor suggestions

since prefetch is Android only, maybe you can use it only for Android? or put it in a try catch since iOS rejects

also, since browser don't have events, the automatic close is not working, I think you can move the close code outside the event listener, maybe with a bigger timeout

File: src/pages/Browser.tsx [hunk 1]
```diff
@@ -6,10 +6,52 @@ import {
...
+    loadedHandler = Browser.addListener('browserPageLoaded', async info => {
+      await timeout(1000);
+      Browser.close();
+      await timeout(200);
+      alert(`The window was closed for you`);
+    });
+  };
+
+  useIonViewDidEnter(() => {
+    Browser.prefetch({ urls: ['https://capacitorjs.com'] });
```
Comment (jcesarmobile) [inline src/pages/Browser.tsx:41]:
```suggestion
```

prefetch was removed, remove usage

File: src/pages/Browser.tsx [hunk 2]
```diff
@@ -6,10 +6,52 @@ import {
...
+    finishedHandler = Browser.addListener('browserFinished', async info => {
```
Comment (jcesarmobile) [inline src/pages/Browser.tsx:42]:
```suggestion
    finishedHandler = Browser.addListener('browserFinished', async () => {
```
doesn't return info anymore

File: src/pages/Browser.tsx [hunk 3]
```diff
@@ -6,10 +6,52 @@ import {
...
+  let finishedHandler: PluginListenerHandle | null = null;
+  let loadedHandler: PluginListenerHandle | null = null;
+
+  function timeout(delay: number) {
+    return new Promise(res => setTimeout(res, delay));
+  }
+
+  const autoCloseAfterDelay = async () => {
+    loadedHandler?.remove();
+    loadedHandler = Browser.addListener('browserPageLoaded', async info => {
```
Comment (jcesarmobile) [inline src/pages/Browser.tsx:32]:
```suggestion
    loadedHandler = Browser.addListener('browserPageLoaded', async () => {
```
doesn't return info anymore

**Comment by jcesarmobile** [comment]
Since the web platform don't have the `browserPageLoaded` event it's not possible to test the close method there

**Review by dwieeb — APPROVED** [review]

**Pull request merged**